libsolidity/analysis/ControlFlowAnalyzer.cpp

```diff
@@ -26,24 +26,50 @@ using namespace std;
 using namespace solidity::langutil;
 using namespace solidity::frontend;
 
+
 bool ControlFlowAnalyzer::analyze(ASTNode const& _astRoot)
 {
+	// Prune reverting paths
+	m_pruner.run(_astRoot);
```

Comment:
This is the only place where `m_pruner` is used, right? Wouldn't it make sense to move this call to CompilerStack.cpp instead?